src/test/java/IMDbBirthdays.java

```diff
@@ -0,0 +1,40 @@
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbBirthdays {
+
+    @Test
+    public void verifyCelebritiesWithTodayBirthday() {
+        open("https://www.imdb.com/feature/bornondate/?ref_=nv_cel_brn");
+
+        SelenideElement bornTodayElement = $x("//span[text()='Born Today']");
+
+        ElementsCollection celebrities = $$("div.sc-7c95f518-3.krmPLY");
```

Comment:
Not a big fan of this element locator, again let's use more reliable stuff (without numbers), like .ipc-metadata-list .krmPLY